src/com/palmergames/bukkit/towny/db/TownyFlatFileSource.java

```diff
@@ -156,11 +156,12 @@ public synchronized boolean backup() throws IOException {
 				FileMgmt.copyDirectory(new File(settingsFolderPath), new File(newBackupFolder));
 				return true;
 			}
-			case "zip": {
-				FileMgmt.zipDirectories(new File(newBackupFolder + ".zip"),
-						new File(dataFolderPath),
-						new File(logFolderPath),
-						new File(settingsFolderPath));
+			case "zip":
```

Comment:
if for w/e reason someone wants to have zip over tar we should still support doing zip files.

Reply:
There's no benefit in using zip over tar.gz actually to be fair there are more disadvantages using the zip format. Hence `tar.gz` should be the standard option.

Reply:
I'm fine with tar being the default but someone might want to still use .zip, so we should continue to support it.

Reply:
I can't come up with a valid reason as to why someone would want that, it takes up more space and it's less universal. Unless there's some argument that shows otherwise I once again don't see why you'd want to support it.

Reply:
"Someone might want to continue using .zip." That's the reason.

Reply:
Yes I know but that reason is moot because tar.gz is better than zip, thus meaning there shouldn't be any reason to *prefer* zip over *tar.gz*

Reply:
I added a "legacyzip" as an option for compression which uses zip.